src/main/java/seedu/address/model/person/Person.java

```diff
@@ -213,6 +219,42 @@ public boolean isSamePerson(Person otherPerson) {
                 && otherPerson.getNric().equals(getNric());
     }
 
+    /**
+     * Returns an immutable set of appointments, which throws
+     * {@code UnsupportedOperationException}
+     * if modification is attempted.
+     */
+    public Set<Appointment> getAppointments() {
+        return Collections.unmodifiableSet(appointments);
+    }
+
+    /**
+     * Adds an appointment to the list of appointments.
+     *
+     * @param appointment The appointment to be added.
+     */
+    public void addAppointment(Appointment appointment) {
+        appointments.add(appointment);
+    }
+
+    /**
```

Comment:
Good job adding clear comments to all the new methods thus far, remember to add the corresponding test cases!